src/integration-test/java/uk/gov/ida/integrationtest/EidasMatchingIntegrationTest.java

```diff
@@ -1,240 +0,0 @@
-package uk.gov.ida.integrationtest;
```

Comment:
Has all of the eIDAS matching functionality been removed from the MSA in this PR or just the tests?